src/main/java/it/academy/app/services/EmailService.java

```diff
@@ -0,0 +1,94 @@
+package it.academy.app.services;
+
+import it.academy.app.models.ApplicationForm;
+import it.academy.app.shared.Constants;
+import org.springframework.stereotype.Service;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Date;
+import java.util.Properties;
+
+@Service
+public class EmailService {
+
+    public EmailService() {
+    }
+
+    public void sendMail(ApplicationForm applicationForm) {
+        Properties props = setupProps();
+        final String username = Constants.EMAIL;
+        final String password = System.getenv("EMAIL_PASS");
+        try {
+            Session session = Session.getDefaultInstance(props,
+                    new Authenticator() {
+                        protected PasswordAuthentication getPasswordAuthentication() {
+                            return new PasswordAuthentication(username, password);
+                        }
+                    });
+            if (applicationForm.getStatus().equals("PERŽIŪRIMA")) {
+                Message message = setupParticipationMessage(session, applicationForm);
+                Transport.send(message);
+            } else {
+                Message message = setupSuccessMessage(session, applicationForm);
+                Transport.send(message);
+            }
+        } catch (MessagingException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Properties setupProps() {
+        final String SSL_FACTORY = "javax.net.ssl.SSLSocketFactory";
+        Properties props = System.getProperties();
+        props.setProperty("mail.smtp.host", "smtp.gmail.com");
```

Comment:
I'd advise keeping configuration in .properties files or other configuration storage.